integration/containerd/cri/integration-tests.sh

```diff
@@ -313,13 +313,21 @@ TestContainerMemoryUpdate() {
 	testContainerStop
 }
 
+# k8s may restart docker which will impact on containerd stop
+stop_containerd() {
+	tmp=$(pgrep kubelet)
```

Comment:
This script is running with `set -e`, so this could cause it to abort unless you do something like:

```suggestion
	local tmp=$(pgrep kubelet || true)
```

Reply:
OK, thanks @jodh-intel.